include/bluetooth/mesh.rst

```diff
@@ -3,9 +3,14 @@
 Bluetooth mesh profile
 ######################
 
-The nRF Connect SDK has experimental support for the Bluetooth mesh specification through the Zephyr :ref:`zephyr:bluetooth_mesh` implementation.
+Bluetooth mesh is supported for development in |NCS|, through the Zephyr :ref:`zephyr:bluetooth_mesh` implementation.
+It's a solution that allows one-to-one, one-to-many, and many-to-many communication, using the Bluetooth Low Energy protocol to exchange messages between the nodes in the network.
```

Comment:
Should we say something about this being published by the Bluetooth SIG, but maybe instead of link to SIG pages we could link to BLE SIG Mesh pages? Or is that not necessary.

Reply:
I added published by Bluetooth SIG.
In our Bluetooth mesh UG, we have several references to SIG's pages. I think it should be sufficient.